config-linux.md

```diff
@@ -8,7 +8,7 @@ The Linux container specification uses various kernel features like namespaces,
 The Linux ABI includes both syscalls and several special file paths.
 Applications expecting a Linux environment will very likely expect these files paths to be setup correctly.
 
-The following filesystems MUST be made available in each application's filesystem
+The following filesystems MUST be made available in each containerized application's filesystem
```

Comment:
You can just say `in each container's filesystem.`

@wking you are using the term incorrectly when referring to namespaces.  A mount namespace is nothing more than a new mounttable for the container and is decoupled from the root filesystem.